src/components/organisms/Header/DesktopHeaderMenu.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+import { Group, HoverCard } from '@mantine/core';
+import { clsx } from 'clsx';
+
+import { ILink } from '@/interfaces/ILink';
+import { Link } from '@/navigation';
+
+import styles from './Header.module.scss';
+
+interface Props {
+  activeLink: string;
+  links: ILink[];
+  onClickLink: (url: string) => void;
+}
+
+const DesktopHeaderMenu: React.FC<Props> = (props) => {
+  const onClickLink = (link: string) => (): void => {
+    props.onClickLink(link);
+  };
+
+  const onClickSubMenuLink =
+    (link: string) =>
+    (e: React.MouseEvent<HTMLAnchorElement>): void => {
+      e.preventDefault();
+
+      props.onClickLink(link);
+    };
+
+  return (
+    <Group visibleFrom="sm" gap="50px">
+      {props.links.map((link) => {
+        const isSubMenuItem = !!link.subLinks;
+        const isSelectedLink = props.activeLink === link.url;
+
+        if (!isSubMenuItem) {
+          return (
+            <Link
+              key={link.label}
+              href={link.url}
+              onClick={onClickLink(link.url)}
```

Comment:
Hm, good point. Initially I thought it would be good to manage selected state only on clicking element for guarantee. I am going try to test this approach, perhaps it would be better and helps to get rid of extra props drilling